lenient/include/module.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * A module for providing utilities for module management.
+ *
+ * To have visibility to `this` and `this_private` of any module during
+ * compilation, 'function context' function definitions are placed behind
+ * macros.
+ *
+ * TODO write what is expected from the file that generates functions using the
+ * macros defined here.
+ *
+ * TODO implement tests.
```

Comment:
Please add issue number where this TODO is to be addressed.

Reply:
Please add number of issue:
https://github.com/pottumuusi/c-encapsulation/issues/3
to the TODO comment.

Reply:
Added in: 588f106b271b2b6927bfef585428636fdb35ea2a